entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+screen -d -m /bin/eatmemory ${RAMINMB}M
```

Comment:
why not just pass the arguments to the docker container? that way you can do this:

```
docker run -d  zoobab/eatmemory:latest 128M
docker run -d  zoobab/eatmemory:latest 5G
```